src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -673,6 +673,27 @@ private bool SaveInternal(string targetFilePath, ProtoCore.Core core)
             return true;
         }
 
+        private void SerializeElementResolver(XmlElement root)
```

Comment:
It's interesting to see this method takes `XmlElement` while `PopulateXmlDocument` takes in `XmlDocument` (because the notion is *saving the map into the document*, not exactly *under the root* because that's a given).